### PR TITLE
NJ/fix: Fixed blog carousel

### PR DESCRIPTION
**Pull request by njazuli-deriv**
Changes:

-   Fixed blog carousel

## Type of change

-   [X] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/academy/components/_hero.js [hunk 1]
```diff
@@ -4,36 +4,30 @@ import styled from 'styled-components'
...
-    height: 100%;
-    object-fit: cover;
 
-    > div {
-        height: 100%;
-    }
-    > div > div {
-        height: 100%;
-    }
+    @media ${device.tablet} {
```
Comment (sean-binary) [inline src/pages/academy/components/_hero.js:22]: Seems like you have 2 `tablet` here.

File: src/components/elements/background-image.js [hunk 2]
```diff
@@ -14,12 +14,12 @@ const StyledBackground = styled(BackgroundImage)`
...
 `
 
-export const Background = ({ children, data, style, dark, ...props }) => {
+export const Background = ({ children, data, fluid, style, dark, ...props }) => {
     const image = getImage(data)
     const bgImage = convertToBgImage(image)
 
     return (
-        <StyledBackground Tag="div" style={style} dark={dark} {...bgImage} {...props}>
+        <StyledBackground Tag="div" style={style} dark={dark} fluid={fluid} {...bgImage} {...props}>
```
Comment (sean-binary) [inline src/components/elements/background-image.js:22]: You shouldn't need to add this. Using the original `data` prop, passing it like usual will work.

File: src/pages/academy/components/_hero.js (hunk 1 above)
Comment (njazuli-deriv) [reply src/pages/academy/components/_hero.js]: amended in the latest commit

File: src/components/elements/background-image.js (hunk 2 above)
Comment (njazuli-deriv) [reply src/components/elements/background-image.js:22]: already tried this but it's not working if we try adding linear-gradient..Im following this guideline https://www.gatsbyjs.com/plugins/gatsby-background-image/#how-to-use-with-multiple-images..

**Review by sean-binary — APPROVED** [review]

**Pull request merged**